### PR TITLE
feat: remove environments

### PR DESCRIPTION
**Pull request by RonenMars**
BREAKING CHANGE: remove the environments from the app according to the backend

## Description
Remove the environment entity from the app after the backend change.

## Linear Ticket
https://linear.app/autokitteh/issue/UI-922/ui-changes-to-not-using-environents

## What type of PR is this? (check all applicable)

-   [x] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/models/user.model.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { User as ProtoUser } from "@ak-proto-ts/users/v1/user_pb";
 import { User } from "@type/models";
 
-export function convertUserProtoToModel(protoUser?: ProtoUser): User {
+export function convertUserProtoToModel(protoUser: ProtoUser): User {
 	return {
-		email: protoUser?.data?.email || "",
-		name: protoUser?.data?.name || "",
+		email: protoUser.email || "",
+		name: protoUser.displayName || "",
```
Comment (efiShtain) [inline src/models/user.model.ts:7]: this is to support non user environment ?

File: src/services/auth.service.ts [hunk 2]
```diff
@@ -11,6 +11,13 @@ export class AuthService {
 	static async whoAmI(): Promise<ServiceResponse<User>> {
 		try {
 			const { user } = await authClient.whoAmI({});
+			if (!user) {
+				throw new Error(
+					i18n.t("userNotFound", {
+						ns: "services",
+					})
+				);
+			}
```
Comment (efiShtain) [inline src/services/auth.service.ts:20]: is this related to this pr?

**Review by efiShtain — APPROVED** [review]

File: src/models/user.model.ts (hunk 1 above)
Comment (RonenMars) [reply src/models/user.model.ts:7]: This is to support the change in the users model in the latest `itay/users` branch.

File: src/services/auth.service.ts (hunk 2 above)
Comment (RonenMars) [reply src/services/auth.service.ts:20]: It's an optimization. 
We changed the user model (https://github.com/autokitteh/web-platform/pull/632/files#diff-20137314f228ed9053b19acb754e4a988a1b87972efb9eda79141369aa737c2b).

Before the change `user` wasn't mandatory, and we had to use conditional value, like this:
`protoUser?.data?.email`

Since the user model has been changed on our side, I optimized the code by a few lines to look better, like this:
`protoUser.email`

Comment (RonenMars) [reply src/services/auth.service.ts:20]: Added the check to pass a user object which is definitely mandatory, and not `undefined` or `null`.
